src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace System.Linq.Expressions.Interpreter
 {
+#if FEATURE_MAKE_RUN_METHODS
```

Comment:
Those flags already exist in the code base. It seems some of those could be beneficial to performance for the interpreter on regular CLR. Right now, they're never enabled by the build. I'm inclined to keep it around and open a separate item to re-evaluate whether those are useful. If not, we can remove the code.